doc/changelog/10-standard-library/10651-new-lemmas-for-lists.rst

```diff
@@ -0,0 +1,3 @@
+- New lemmas for lists.
```

Comment:
Could you please be a bit more specific: new lemmas on `combine`, `filter`, and `nodup` functions on lists. The lemma `filter_app` was moved to the `List` module.